webforces/server/mongodbworker.py

```diff
@@ -312,6 +312,29 @@ def getAllAlgs(self) -> Tuple[DBStatus, List[Algorithm]]:
         logger.debug("Algorithms was found")
         return (DBStatus.s_ok, algs)
 
+    def getAllAlgsWithFlags(self, user_id) -> Tuple[DBStatus, List[Algorithm], List[bool]]:
```

Comment:
If you use 0 and 1 as flag values then it's not `List[bool]`, it's `List[int]`
You need either change type hints or change values.
Note: if you change type hints don't forget to update them in `DBWorker`.

Reply:
Replaced:
1 -> True
0 -> False